resources/css/style.css

```diff
@@ -22,19 +22,24 @@
 .animated-accordion-container .accordion-item h3 {
 	padding: 0;
 	margin: 0;
+	font-size: 24px;
+    font-weight: bold;
 }
 
 
 .animated-accordion-container .accordion-item.active, 
 .animated-accordion-container .accordion-item.active h3 {
 	color: #fff;
+	font-size: 24px;
```

Comment:
You dont need to repeat the `font-size` and `font-weight`

Reply:
Did you fix the border? 

https://github.com/praem90/animated-accordion-elementor-widget/issues/1

Reply:
We can fix it in separate PR